content/guides/testing/introduction.md

```diff
@@ -218,6 +220,34 @@ export const runnerHooks: Required<Pick<Config, 'setup' | 'teardown'>> = {
 }
 ```
 
+#### Truncate database after each run cycle
+
+An alternative to the above approach is to truncate all table in the database after each run cycle instead of roll it back. This can be done by registering the `TestUtils.db().truncate()` hook within the `tests/bootstrap.ts` file.
```

Comment:
```suggestion
An alternative to the above approach is to truncate all tables in the database after each run cycle instead of rolling it back. This can be done by registering the `TestUtils.db().truncate()` hook within the `tests/bootstrap.ts` file.
```